mednafen/pce_fast/vdc.cpp

```diff
@@ -849,7 +843,11 @@ void VDC_RunFrame(EmulateSpecStruct *espec, bool IsHES)
          };
 
          DisplayRect->x = 0;
-         DisplayRect->w = ws[correct_aspect][vce.dot_clock];
+         if(hoverscan == 1 && vce.dot_clock == 1){
+			   DisplayRect->w = 352;
+		   }else{
+			   DisplayRect->w = ws[correct_aspect][vce.dot_clock];
```

Comment:
We should make sure we understand which array is always used based on this: https://github.com/westonlast/beetle-pce-fast-libretro/blob/master/mednafen/settings.cpp#L31

Well, ideally I should remove it to complete this commit: 1a190a6